web/src/components/apps/Dashboard.tsx

```diff
@@ -634,6 +733,9 @@ class Dashboard extends Component {
     const appResourcesByState = this.getAppResourcesByState();
     const hasStatusInformers = this.checkStatusInformers();
 
+    // TODO: remove the type casting
```

Comment:
```suggestion
```

Reply:
just going to leave this in for now. i used this to try to figure out an issue. will refactor when we convert to SFC and use hooks.